.github/workflows/release.yml

```diff
@@ -0,0 +1,104 @@
+name: Release
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "Tag name, e.g. 0.7.11"
+        default: ""
+        required: true
+
+jobs:
+  build:
+    name: Upload Release Asset
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Login to Quay.io
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ vars.IMAGE_REGISTRY }}
+          username: ${{ secrets.docker_username }}
+          password: ${{ secrets.docker_password }}
+
+      - name: Git set user
+        shell: bash
+        run: |
+          git config user.name "$USERNAME"
+          git config user.email "$USERENAME-bot@users.noreply.github.com"
+        env:
+          USERNAME: ${{ github.actor }}
+
+      - name: Update the VERSION
+        run: |
+          echo "$VERSION" > VERSION
+        env:
+          VERSION: ${{ github.event.inputs.tag }}
+
+      # - name: Build model-server-base
+      #   run: |
+      #     make build-base
+      #   env:
+      #     IMAGE_REGISTRY: ${{ vars.IMAGE_REGISTRY }}
+
+      # - name: Push model-server-base
+      #   run: |
+      #     make push-base
+      #   env:
+      #     IMAGE_REGISTRY: ${{ vars.IMAGE_REGISTRY }}
+
+      # - name: Update base in model-server
+      #   run: |
+      #     sed -i "s/model_server_base:.*/model_server_base:v$VERSION/g" ./dockerfiles/Dockerfile
+      #   env:
+      #     VERSION: ${{ github.event.inputs.tag }}
```

Comment:
@sthaha @sunya-ch Do we also want to release a new version of the model-server-base at the time or making model-server release? Or do we want to release it separately?